pkg/log/logger_test.go

```diff
@@ -62,8 +63,9 @@ func TestIsDebugEnabled(t *testing.T) {
 		Encoding: "console",
 	}
 	var err error
-	logger, err = zapConfig.Build()
+	logger, err := zapConfig.Build()
 	assert.NilError(t, err, "debug level logger create failed")
+	InitializeLogger(logger, &zapConfig)
```

Comment:
These tests explicitly did not trigger the `once.Do()` to make sure we re-try things.

Reply:
re-init is no longer a problem as we handle this case cleanly.

Reply:
Per offline discussions, reverting all logging changes.